continuous_integration/environment-3.8.yaml

```diff
@@ -5,7 +5,7 @@ channels:
 dependencies:
   # required dependencies
   - python=3.8.*
-  - numpy=1.17.*
+  - numpy=1.19.*
```

Comment:
What do you think about using `numpy=1.20.*` and just `xfail`ing `test_futures_to_delayed_dataframe` for when `numpy >= 1.20`? Or are there other NumPy 1.20 compatibility issues?

Reply:
It seemed like a flaky test, so I'd probably skip it instead of failing, but yeah that is fine with me.

Reply:
Ah, nvm then. I hadn't realized it was a flaky test and not failing consistently